templates/results-row.php

```diff
@@ -14,5 +14,10 @@
 	<td>
 		{{data.message}}
 	</td>
+	<td>
+		<# if ( data.link ) { #>
+			<a href="{{data.link}}" title="<?php _e( 'View file in the plugin file editor.', 'plugin-check' ); ?>" aria-label="<?php _e( 'View file in the plugin file editor.', 'plugin-check' ); ?>" target="_blank"><?php _e( 'View in code editor', 'plugin-check' ); ?></a>
```

Comment:
`title` attribute is bad for a11y, let's remove it.

Reply:
Thanks for the review. Removed in https://github.com/WordPress/plugin-check/pull/298/commits/f323f66f638430e109ec07c63545a27fba93dae7